gcode/extrusion-tracker.go

```diff
@@ -62,15 +63,13 @@ func (et *ExtrusionTracker) TrackInstruction(instruction Command) {
 	} else if instruction.IsSetPosition() {
 		hasParamsOrFlags := len(instruction.Params) > 0 || len(instruction.Flags) > 0
 		if hasParamsOrFlags {
+			et.LastCommandWasG92 = true
```

Comment:
Not technically correct to move this out, as the if/else-if/else-if doesn't have an else case, and there are in fact G92 commands that don't affect the E axis (or the A/B axes). For example, the following shouldn't set the `LastCommandWasG92` flag:

```gcode
G92 X0 ; set current X position to 0
```